### DISCUSSION TITLE
Saleor With Next.JS store front cookie problem

### DISCUSSION THREAD

**Discussion by sadegh-khan** [saleor/saleor#13383] [Q&A]
hi,
i have used Next.js for storeFront in different site, my problem is that set-cookie is ok and sent but not saved in cookie!?
only different is Next.js and React.JS allthing are same!

same-site =Lax
Secure = True 

also with samesite = 'None' and secure = True , cookies not save!

what is wrong? can anyone help me :)

**Comment by timuric** [comment]
hi @sadegh-khan are you using saleor storefront? Which auth library is being used?

**Reply by sadegh-khan** [reply]
no, i am begin to write my own storefront in next.js
in back i am using saleor default auth ()
